### DISCUSSION TITLE
Increase the specificity of ALL Tailwind Classes (layers?)

### DISCUSSION THREAD

**Discussion by npham-dev** [tailwindlabs/tailwindcss#13057] [Help] [answered]
I have a custom component library that uses [postcss-cascade-layers](https://www.npmjs.com/package/@csstools/postcss-cascade-layers), meaning the classes within the component library will look something like this: `.class:not(#\#)` when built (to increase specificity). 

I know `!utility` will make the utility more "important", but I am wondering if there is a way to make ALL Tailwind classes have a higher default precedence (such as adding a suffix like multiple `:not`) such that my component library doesn't override the Tailwind styles.

**Comment by wongjn** [comment, marked as answer]
You could look at setting the [`important`](https://tailwindcss.com/docs/configuration#important) option if all your Tailwind classes are utility classes only. Otherwise you could look at a custom PostCSS plugin to make them have higher specificity.

**Reply by npham-dev** [reply]
The first option is exactly what I was looking for (it's not "fine grained" but it works well enough). Thanks!
